pom.xml

```diff
@@ -195,6 +200,14 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.18.1</version>
+          <dependencies>
+            <dependency>
+              <!-- Force junit provider because surefire will choose the testng one if testng is detected -->
+              <groupId>org.apache.maven.surefire</groupId>
+              <artifactId>surefire-junit47</artifactId>
```

Comment:
@Tibor17 Is it possible to use the junit provider for some tests and the testng provider for some other tests in the same module?
As workaround, I had to create a dedicated module :(